src/chart/Bubble.js

```diff
@@ -35,6 +36,23 @@
         return retVal;
     };
 
+    Bubble.prototype.enter = function () {
+        if (this.useTooltip()) {
+            if (!this._parentOverlay && this._parentWidget._parentOverlay) {
+                this._parentOverlay = this._parentWidget._parentOverlay;  //This is because charts do not get an overlay div when they are inside the MultiChart
+            }
+            this._parentOverlay.selectAll(".layout_Tooltip").each(function(d) {
+                d.node().parentNode.remove();
+            });
+            
+            this._tooltip = new Tooltip()
+                .registerWidget(this)
+                .target(this._parentOverlay.node())
+                .render()
+            ;
+        }
+    };
+    
```

Comment:
Agreed, and on that note i think a lot of the charts that inherit scatter should actually have that scatter code placed in a "common" file instead of inheriting directly from scatter from an organizational POV.